src/components/grid-aware/ThreeParagraphBlock/ThreeParagraphBlock.js

```diff
@@ -19,16 +33,39 @@ const ImagePropType = PropTypes.shape({
 
 /* Subcomponents */
 
-const ParagraphBlock = ({ title, description }) => (
-  <div>
-    <div className={s.paragraphTitle}>{title}</div>
-    <div className={s.paragraphDescription}>{description}</div>
-  </div>
-);
+const ParagraphBlock = ({ title, description, button }) => {
+  let hasButton;
+
+  if (button) {
+    hasButton = (
+      <div className={s.ctaButton}>
```

Comment:
Since there are several different kinds of CTA buttons on the page, how about we give this CSS class a more specific name, like `paragraphCTAButton`, to make it easier to differentiate between the different instances?